dbgap_extract.py

```diff
@@ -138,15 +159,35 @@ def get_previous_version_of_study_accession(study_accession):
     return previous_version_of_study_accession
 
 
-def get_sample_use_from_xml_sample(sample):
+def _get_flattened_sample_use_from_xml_sample(sample):
```

Comment:
this function is unused

Reply:
yeah, I wanted to leave it around in case we ever want to support using this format instead of the new format. but I could just remove and we can always check the git history? :thinking:

Reply:
i dont mind if you leave it in =] just note that in the docstring?

Reply:
Imma remove it for now :+1:

Reply:
I think you're right, better to remove unused code and just add it later if we need it